_pages/qa/aloha-request-leave-vacation-sick-time.md

```diff
@@ -27,7 +27,7 @@ Yes, it's preferred if you can submit it as soon as possible.
 ## How do I request time off?
 
 1. Visit [ALOHA](http://aloha.gsa.gov/) and use your GSA credentials to log in.
-2. Submit your supervisor approval to Aaron Snow.
+2. Submit your supervisor approval to your supervisor.
```

Comment:
it is possible that I am misunderstanding this -- in ALOHA there is a field for indicating supervisor. I put Aaron there. Is there a separate step where we submit the ALOHA approval to Aaron? If so, how does that work, and do we need to get confirmation from him?